plugins/base/src/main/kotlin/signatures/JvmSingatureUtils.kt

```diff
@@ -0,0 +1,129 @@
+package org.jetbrains.dokka.base.signatures
+
+import javaslang.Tuple2
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+
+interface JvmSingatureUtils {
+
+    fun PageContentBuilder.DocumentableContentBuilder.annotationsBlock(d: Documentable)
```

Comment:
I think all those methods can be private to not leak outside the implementing type

Reply:
Private members in an interface?